docs/rules/readonly-array.md

```diff
@@ -29,29 +31,39 @@ const points: ReadonlyArray<Point> = [{ x: 23, y: 44 }];
 points.push({ x: 1, y: 2 }); // Unresolved method push()
 ```
 
-#### Has Fixer
+## Options
 
-Yes
+The rule accepts an options object with the following properties:
 
-#### Options
+```typescript
+type Options = {
+  readonly ignoreReturnType?: boolean;
+  readonly ignoreLocal?: boolean;
+  readonly ignorePattern?: string | Array<string>;
+  readonly ignorePrefix?: string | Array<string>;
+  readonly ignoreSuffix?: string | Array<string>;
+};
+
+const defaults = {
+  ignoreReturnType: false,
+  ignoreLocal: false
+};
+```
 
-- [ignore-local](#using-the-ignore-local-option)
-- [ignore-prefix](#using-the-ignore-prefix-option)
-- [ignore-suffix](#using-the-ignore-suffix-option)
-- [ignore-pattern](#using-the-ignore-pattern-option)
-- [ignore-return-type](#using-the-ignore-return-type-option)
-- [ignore-rest-parameters](#using-the-ignore-rest-parameters-option)
+### `ignore-rest-parameters`
 
-#### Example config
+TODO: This option does not seem to exist in the code?
```

Comment:
I removed this option as it is no longer needed. Rest parameters can now be declared as readonly arrays.
https://github.com/Microsoft/TypeScript/issues/15972

Reply:
👍 Then we can remove it from the docs.